docs/scos/dev/tutorials-and-howtos/introduction-tutorials/tutorial-sending-an-email.md

```diff
@@ -45,11 +45,6 @@ class Customer extends SprykerCustomer
         // used for further processing
         $mailTransfer = new MailTransfer();
 
-        // Set the mail type which is used for the
-        // internal mapping—for example, which mail provider
-        // should send this mail
-        $mailTransfer->setType(CustomerRegistrationMailTypePlugin::MAIL_TYPE);
-
```

Comment:
Why it was removed?
We still use `MailTransfer::TYPE`, but with mail type builders.

Reply:
Yep, but it's set now in `MailTypeBuilderPlugin`, there is no reason to set twice

Reply:
It's possible to send emails directly from the controller or the business model, without having `MailType(Builder)` plugin. (It's an example)
I so it's different.
Please revert it, and modify `mailType` => `mailTypeBuilder`